### PR TITLE
Remove gateway, save ui, bump webpack

### PR DESCRIPTION
**Pull request by carlioth**
Closes qlik-oss/core-assisted-prescription#440

We now only host the ui in this repo. 
Bump to use webpack 4, and Netlify added for building the site.
This pr will disable auth for the site. Enabling a new version of Auth is described / handled in https://github.com/qlik-oss/core-assisted-prescription/issues/441

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,11 @@
-version: "3.1"
+version: "2"
```
Comment (wennmo) [inline docker-compose.yml]: Any reason for lowering the docker-compose file version?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,19 +1,11 @@
...
     ports:
-      - "443:443"
-    volumes:
-      - ../info/site:/usr/local/openresty/nginx/html/docs
-      - ./custom-analytics-ui/dist:/usr/local/openresty/nginx/html/custom-analytics-ui
-
-networks:
-  core-assisted-prescription_default:
-    external: true
+      - "9176:9076"
```
Comment (wennmo) [inline docker-compose.yml]: Is this file used anywhere e.g for cci testing or is it just for local development?

**Comment by hrigner** [comment]
The inner custom-anlytics-ui folder should be removed..

File: docker-compose.yml (hunk 1 above)
Comment (carlioth) [reply docker-compose.yml]: Bumped it to 3 again.

File: docker-compose.yml (hunk 2 above)
Comment (carlioth) [reply docker-compose.yml]: It's for local development

**Comment by carlioth** [comment]
> The inner custom-anlytics-ui folder should be removed..

Folder is removed, probably a left over from earlier checkouts.

File: src/main.jsx [hunk 3]
```diff
@@ -21,43 +21,41 @@ import './main.css';
...
+  //   fetch('/logout', {
+  //     credentials: 'same-origin',
+  //   }).then(cb);
+  // },
 };
 
 // Main component responsible for rendering the routes when
 // the path matches the route.
-const Main = ({ isAuthenticated, notAuthorizedCallback }) => (
+const Main = ({ notAuthorizedCallback }) => (
```
Comment (hrigner) [inline src/main.jsx]: Keep the isAuthenticated as prop and use as prop to PrivateRoute. We initialise it to false so no worries.

File: src/main.jsx [hunk 4]
```diff
@@ -21,43 +21,41 @@ import './main.css';
...
   <main>
     <Switch>
       <Route exact path="/" component={LandingPage} />
-      <PrivateRoute path="/app" component={App} isAuthenticated={isAuthenticated} notAuthorizedCallback={notAuthorizedCallback} />
```
Comment (hrigner) [inline src/main.jsx]: Keep isAuthenticated={isAuthenticated}

File: src/main.jsx [hunk 5]
```diff
@@ -21,43 +21,41 @@ import './main.css';
...
+      <PrivateRoute path="/app" component={App} isAuthenticated notAuthorizedCallback={notAuthorizedCallback} />
     </Switch>
   </main>
 );
 
 Main.propTypes = {
-  isAuthenticated: PropTypes.bool.isRequired,
```
Comment (hrigner) [inline src/main.jsx]: And keep this one

**Pull request merged**